### ISSUE TITLE
`make install` installs in system directories when configured otherwise

### ISSUE THREAD

**Issue by egarrulo** [eranif/codelite#1645]
Hello,

even when after compiling with the option to install all files in a non-system directory, installation fails after trying to write in system directories:

$ cmake -G "Unix Makefiles" -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX:PATH=/opt/codelite/codelite-10.0 ..
$ make
$ make install
[skipped]
Install the project...
-- Install configuration: ""
CMake Error at sdk/wxsqlite3/cmake_install.cmake:50 (file):
  file cannot create directory: /usr/lib/x86_64-linux-gnu/codelite.  Maybe
  need administrative privileges.
Call Stack (most recent call first):
  cmake_install.cmake:37 (include)

Makefile:85: recipe for target 'install' failed
make: *** [install] Error 1

**Comment by dghart** [comment]
Hi,

CodeLite has always used CL_PREFIX to set the install destination.

In the past CMAKE_INSTALL_PREFIX would probably have worked too. However a couple of years ago it had to be disabled for the following reason (taken from the top CMakeLists.txt):

```
if ( NOT CL_PREFIX )
    # If the caller hasn't set his own destination, install to a multi-arch lib dir if applicable
    if (CMAKE_VERSION VERSION_GREATER 2.8.7 AND ( UNIX AND NOT APPLE ))
        if (CMAKE_VERSION VERSION_GREATER 3.0.0)
            # Prior to this, afaict the GNUInstallDirs module worked whatever the prefix
            # Since, it looks at CMAKE_INSTALL_PREFIX which is /usr/local by default, and refuses to run the multiarch-setting code unless it's /usr/
            # So, partly to comply with the default documented above, & partly for packaging, explicitly set it to /usr
            set (CMAKE_INSTALL_PREFIX "/usr")
        endif()
        include (GNUInstallDirs) # defines CMAKE_INSTALL_LIBDIR to lib or lib64 or whatever. Available since cmake 2.8.8
        set( CL_INSTALL_LIBDIR ${CMAKE_INSTALL_LIBDIR} )
    endif()
    
    set( CL_PREFIX "/usr" )
endif(NOT CL_PREFIX)
```
I suggest you try instead:
 ```
$ cmake -G "Unix Makefiles" -DCMAKE_BUILD_TYPE=Release -DCL_PREFIX=/opt/codelite codelite-10.0 ..
```
which should work if you have write access to /opt.

Regards,

David

**Comment by egarrulo** [comment]
Your suggestion works, thanks.
